webserver/__init__.py

```diff
@@ -10,6 +10,10 @@ def create_postgres():
     from postgres_connection import init_postgres_connection
     return init_postgres_connection(current_app.config['SQLALCHEMY_DATABASE_URI'])
 
+def create_redis():
```

Comment:
@alastair @gentlecat i did some changes to create single connections for `redis` (RedisListenStore) and `postgres` (PostgresListenStore). It might need some more changes.
Changes: https://github.com/metabrainz/listenbrainz-server/pull/92/commits/b6a17c890d58cb9d6254b36277252d75f8ed1f1f